drkafka/src/main/java/com/pinterest/doctorkafka/api/DecommissionBrokerApi.java

```diff
@@ -0,0 +1,53 @@
+package com.pinterest.doctorkafka.api;
+
+import com.pinterest.doctorkafka.DoctorKafka;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+
+@Path("/cluster/{clusterName}/broker/{brokerId}/admin/decommission")
+@Produces({MediaType.APPLICATION_JSON })
+@Consumes({MediaType.APPLICATION_JSON })
+public class DecommissionBrokerApi extends DoctorKafkaApi {
```

Comment:
Can we dump add an API to list (GET) all decommissioned brokers?

Reply:
Decommission status is now exposed in the GET /api/clusters/{clusterName}/brokers endpoint response